src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -589,7 +596,8 @@ class BinaryCompactObject : public DomainCreator<3> {
       double outer_radius_domain,
       const typename InitialRefinement::type& initial_refinement,
       const typename InitialGridPoints::type& initial_number_of_grid_points,
-      bool use_projective_map = true, double frustum_sphericity = 0.0,
+      bool use_equiangular_map = true, bool use_projective_map = true,
+      double frustum_sphericity = 0.0,
```

Comment:
Minor comment here and for line 626: Now that we have equiangular capability, and the frustum layer can be made spherical, should this default value for the `frustum_sphericity` be kept at `0.0`?